### ISSUE TITLE
rename package in go.mod

### ISSUE THREAD

**Issue by miki725** [lmika/dynamo-browse#50]
```
➜ go install github.com/lmika/dynamo-browse/cmd/dynamo-browse@latest
go: github.com/lmika/dynamo-browse/cmd/dynamo-browse@latest: github.com/lmika/dynamo-browse@v0.2.0: parsing go.mod:
        module declares its path as: github.com/lmika/audax
                but was required as: github.com/lmika/dynamo-browse
```

**Comment by lmika** [comment]
Ah, whoops! Forgot about fixing that. Thanks for the report. Will address in the next PR.

**Comment by lmika** [comment]
Fixed and merged in PR #52
